hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/SchemaRegistryProvider.java

```diff
@@ -52,13 +53,34 @@ public static class Config {
     public static final String SRC_SCHEMA_REGISTRY_URL_PROP = "hoodie.deltastreamer.schemaprovider.registry.url";
     public static final String TARGET_SCHEMA_REGISTRY_URL_PROP =
         "hoodie.deltastreamer.schemaprovider.registry.targetUrl";
+    public static final String SCHEMA_CONVERTER_PROP = "hoodie.deltastreamer.schemaprovider.registry.schemaconverter";
+  }
+
+  @FunctionalInterface
+  public interface SchemaConverter {
+    /**
+     * Convert original schema string to avro schema string.
+     *
+     * @param schema original schema string (e.g., JSON)
+     * @return avro schema string
+     */
+    String convert(String schema) throws IOException;
+  }
+
+  public Schema parseSchemaFromRegistry(String registryUrl) throws IOException {
```

Comment:
`TestJsonToAvroSchemaConverter` already covers the conversion part. To cover this method in test and verify config works properly, we can add one small test with json schema to `TestSchemaRegistryProvider`.